addons/core/XEH_postInit.sqf

```diff
@@ -2,6 +2,9 @@
 
 if (!isMultiplayer && !is3DENMultiplayer) exitWith {}; //Don't do anything in Singleplayer
 
+if(isServer) then {
+    {_x call TFAR_fnc_processRadiosServer } forEach playableUnits;
```

Comment:
no space before `}` and maybe a `;`

Reply:
Why are you including not occupied Units?

Reply:
There is no reason to not replace radios for not occupied playble units. It reduces traffic for joined in progess players.

Reply:
The problem you are trying to fix was the high initial traffic when the mission starts and tons of players join. And now you say it doesn't matter?

Reply:
No. I didn't say that.  I said that repalce redios for all playable units have positive performance impact for JIP.

Reply:
But negative impact at non-jip mission start.